_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -183,7 +183,23 @@ And on Debian/Ubuntu installations with Passenger, run:
 
     service apache2 restart
 
-#### Optional Step 5 - Cleanup of RPMs and repositories
+#### Optional Step 5 - Reclaim database space
+
+After database migrations, some space can sometimes be reclaimed. It's good idea to perform database vacuum for postgres on a regular basis as well rather than relying on autovacuum feature. For PostgreSQL do the following:
+
+    foreman-maintain service stop
+    service postgresql start
+    su - postgres -c 'vacuumdb -f -d foreman'
+    foreman-maintain service start
+
+MariaDB/MySQL users should do the similar:
+
+    foreman-maintain service stop
+    service mysql start
+    mysqlcheck -o --all-databases
```

Comment:
In documentation I'd always advise long options so `--optimize`

Reply:
Good point will do the same for pg.